src/node_process_methods.cc

```diff
@@ -471,6 +473,128 @@ static void ReallyExit(const FunctionCallbackInfo<Value>& args) {
   env->Exit(code);
 }
 
+inline char** copy_js_strings_array(Environment* env,
+                                    Local<Array> js_array,
+                                    int* target_length) {
+  Local<Context> context = env->context();
+  char** target = nullptr;
+  int length = js_array->Length();
+
+  CHECK_LT(length, INT_MAX);
+
+  target = new char*[length + 1];
+  target[length] = nullptr;
+
+  if (length > 0) {
+    for (int i = 0; i < length; i++) {
+      node::Utf8Value str(env->isolate(),
+                          js_array->Get(context, i).ToLocalChecked());
+      target[i] = strdup(*str);
```

Comment:
you can just use std::memcpy and write modern C++. strdup is a C API.

Reply:
Fixed, thanks!